src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -0,0 +1,208 @@
+
+  const tabItems = VALUES.filter((_, index) => index === NETWORK_INDEX)
+
+  const container = useRef()
+  useEffect(() => {
+    if (container.current) {
+      container.current.focus()
+    }
+  }, [])
+
+  return (
+    <div ref={container} tabIndex="0" css="outline: 0">
+      <Layout css="z-index: 2">
+        <Close compact={compact} onClick={onClose} />
+        <Header
+          primary="Global preferences"
+          css={`
+            padding-top: ${!compact ? 10 * GU : 0}px;
+          `}
+        />
+        <React.Fragment>
+          {tabItems.length > 1 ? (
+            <Tabs
+              items={tabItems}
+              onChange={onNavigation}
+              selected={sectionIndex}
+            />
+          ) : (
+            <Bar>
+              <div
+                css={`
+                  display: flex;
+                  height: 100%;
+                  align-items: center;
+                  padding-left: ${compact ? 2 * GU : 3 * GU}px;
+                  color: ${compact
+                    ? theme.surfaceContent
+                    : theme.surfaceContentSecondary};
+                  ${textStyle('body2')}
+                `}
+              >
+                {tabItems[0]}
+              </div>
+            </Bar>
+          )}
+          <main>{sectionIndex === NETWORK_INDEX && <Network />}</main>
+        </React.Fragment>
+      </Layout>
+    </div>
+  )
+}
+
+function useGlobalPreferences({ path = {}, onScreenChange }) {
+  const [sectionIndex, setSectionIndex] = useState(null)
+  const [subsection, setSubsection] = useState(null)
+  const handleNavigation = useCallback(
+    index => {
+      onScreenChange(PATHS[index])
+    },
+    [onScreenChange]
+  )
+
+  useEffect(() => {
+    if (!path) {
+      setSectionIndex(null)
+      return
+    }
+    const index = PATHS.findIndex(item => path.startsWith(item))
+
+    if (index !== NETWORK_INDEX) {
+      return
+    }
+    setSectionIndex(index === -1 ? null : index)
+
+    // subsection is the part after the PATH, e.g. for `?p=/notifications/verify` - `/verify`
+    const subsection = index !== -1 ? path.substring(PATHS[index].length) : null
+
+    setSubsection(subsection)
+    // Does the current path start with any of the declared route paths
+  }, [path, sectionIndex])
+
+  return { sectionIndex, subsection, handleNavigation }
+}
+
+function Close({ compact, onClick }) {
+  const theme = useTheme()
+  return (
+    <div
+      css={`
+        position: absolute;
+        right: 0;
+        padding-top: ${2.5 * GU}px;
+        padding-right: ${3 * GU}px;
+
+        ${compact &&
+          `
+            padding-top: ${2 * GU}px;
+            padding-right: ${1.5 * GU}px;
+          `}
+      `}
+    >
+      <ButtonIcon onClick={onClick} label="Close">
+        <IconClose
+          css={`
+            color: ${theme.surfaceIcon};
+          `}
+        />
+      </ButtonIcon>
+    </div>
+  )
+}
+
+function AnimatedGlobalPreferences(props) {
```

Comment:
```suggestion
function AnimatedGlobalPreferences({ onScreenChange, path }) {
```

And then we can remove the `props.` usage later.